packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -15,6 +15,7 @@ import { jsonArrayContainsSelector } from '../../utils/viewUtils';
 
 export const DEFAULT_LOW_KARMA_THRESHOLD = -10
 export const MAX_LOW_KARMA_THRESHOLD = -1000
+export const EAF_COMMUNITY_TOPIC_ID = 'ZCihBFp5P64JCvQY6'
```

Comment:
I would slightly prefer this to be a db setting and/or an array, if only to make it easier to test on dev. Not important though